signals/commands/devops/src/main/java/org/eclipse/ditto/signals/commands/devops/RetrieveStatisticsDetails.java

```diff
@@ -42,18 +49,37 @@
      */
     public static final String TYPE = TYPE_PREFIX + NAME;
 
-    private RetrieveStatisticsDetails(final DittoHeaders dittoHeaders) {
+    private final List<String> shardRegions;
+    private final List<String> namespaces;
+
+    private RetrieveStatisticsDetails(final List<String> shardRegions, final List<String> namespaces,
+            final DittoHeaders dittoHeaders) {
         super(TYPE, null, null, dittoHeaders);
+        this.shardRegions = shardRegions;
```

Comment:
The class isn't immutable when we just assign the lists. Please use pattern `Collections.unmodifiableList(new ArrayList(shardRegions))`

Reply:
good point